cybersecurity/port-scanner/devashmit-day17/port_scanner.py

```
@@ -0,0 +1,39 @@
+          143:"IMAP",443:"HTTPS",445:"SMB",3306:"MySQL",3389:"RDP",5432:"PostgreSQL",
+          6379:"Redis",8080:"HTTP-Alt",8443:"HTTPS-Alt",27017:"MongoDB"}
+open_ports=[]; lock=threading.Lock()
+
+def scan(target, port):
+    try:
+        s=socket.socket(socket.AF_INET,socket.SOCK_STREAM); s.settimeout(0.5)
+        if s.connect_ex((target,port))==0:
+            svc=SERVICES.get(port,"Unknown")
+            with lock: open_ports.append((port,svc)); print(f"  [OPEN] {port:<6} {svc}")
+        s.close()
+    except: pass
```

Ruff (0.15.15)

[error] Do not use bare `except`

(E722)